src/ucs/config/parser.h

```diff
@@ -127,9 +135,12 @@ typedef struct ucs_config_bw_spec {
         .table  = _table, \
         .name   = _name, \
         .prefix = _prefix, \
-        .size   = sizeof(_type) \
+        .size   = sizeof(_type), \
+        .flags  = 0 \
     };
 
+#define UCS_CONFIG_GET_TABLE(_table) &_table##_config_entry
```

Comment:
2 space lines in H file

Reply:
`GET_TABLE` returns `_entry`. Maybe rename to smth like `_GET_(CONFIG)_ENTRY`

Reply:
GET_TABLE is consistent with some other macros: ADD_TABLE, REMOVE_TABLE.
shouldn't it remain consistent?